subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_firmware.c

```diff
@@ -460,6 +484,22 @@ static void update_process(void)
 		reboot_work_handler();
 	}
 }
+#endif
+
+static uint8_t apply_firmware_delta_modem_update(void)
```

Comment:
Who is calling this static function?

Reply:
Thanks, I've updated with your changes now.